java/arcs/android/demo/DemoService.kt

```diff
@@ -25,31 +26,19 @@ import kotlinx.coroutines.withContext
 /**
  * Service which wraps an ArcHost.
  */
-class DemoService : LifecycleService() {
+class DemoService : ArcHostService() {
 
     private val coroutineContext = Job() + Dispatchers.Main
-    private val scope = CoroutineScope(coroutineContext)
 
     private lateinit var notificationManager: NotificationManager
 
-    private val myHelper: ArcHostHelper by lazy {
-        val host = MyArcHost(
-            this,
-            this.lifecycle,
-            JvmSchedulerProvider(coroutineContext),
-            ::ReadPerson.toRegistration(),
-            ::WritePerson.toRegistration()
-        )
-        ArcHostHelper(this, host)
-    }
-
-    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
-        super.onStartCommand(intent, flags, startId)
-
-        myHelper.onStartCommand(intent)
-
-        return START_NOT_STICKY
-    }
+    override val arcHost = MyArcHost(
+        this,
+        this.lifecycle,
+        JvmSchedulerProvider(coroutineContext),
+        ::ReadPerson.toRegistration(),
+        ::WritePerson.toRegistration()
+    )
 
```

Comment:
add

https://github.com/PolymerLabs/arcs/blob/ea8e06652a8de0ff08ff7132b6e2ef4663669af6/java/arcs/android/sdk/host/ArcHostService.kt#L32

val myArcHost = MyArcHost(
        this,
        this.lifecycle,
        JvmSchedulerProvider(coroutineContext),
        ::ReadPerson.toRegistration(),
        ::WritePerson.toRegistration()
    )

override val arcHosts: List<ArcHost> by lazy { listOf(myArcHost) }

I'm going to remove the 'arcHost' field eventually.